crates/sui/src/sui_commands.rs

```diff
@@ -404,11 +404,14 @@ async fn prompt_if_no_config(wallet_conf_path: &Path) -> Result<(), anyhow::Erro
 
             println!("Do you want to generate a secp256k1 keypair instead? [y/N] No will select Ed25519 by default. ");
             let key_scheme = match read_line()?.trim() {
-                "y" => Some(String::from("secp256k1")),
-                _ => None,
+                "y" => SignatureScheme::Secp256k1,
+                _ => SignatureScheme::ED25519,
```

Comment:
This works for now, but in the future I'd prefer we specifically request for 0 and 1 (instead of y/N). This is because 0, 1 can extend to support multiple key types (and y/N) is just a temp solutions (it happens to work just because we have two schemes only - but we'll soon might add seck256r1 (NIST curve))